src/input.js

```diff
@@ -21,6 +22,7 @@ class Input extends Component {
   constructor(props, context) {
     super(props, context)
     this.autocomplete = this.context[AUTOCOMPLETE_CONTEXT]
+    this.autoFocus = typeof this.context[MENU_CONTEXT] !== 'undefined'
```

Comment:
I _think_ this should be fine since this will only evaluate true if it's a child of `Menu` which means `Menu` needs to render first right?